python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
@ueshin , the change in #18933 seems to be to just fix this test and I think it makes things more confusing.  Also, while that new conf would allow `test_toPandas_arrow_toggle` to pass, `test_pandas_round_trip` would still fail without having to de-localize the timestamps.